bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/Handle.java

```diff
@@ -55,6 +55,17 @@ default void close() throws BKException, InterruptedException {
         FutureUtils.result(asyncClose());
     }
 
+    /**
+     * Returns the metadata of this ledger.
+     *
+     * <p>This call only retrieves the metadata cached locally. If there is any metadata updated, the read
+     * handle will receive the metadata updates and update the metadata locally. The metadata notification
+     * can be deplayed, so it is possible you can receive a stale copy of ledger metadata from this call.
+     *
+     * @return the metadata of this ledger.
+     */
+    LedgerMetadata getLedgerMetadata();
```

Comment:
rename to getMetadata()? The ledger bit is redundant.

Reply:
there is already a "getLedgerMetadata()" in LedgerHandle. it is easier to just provide the same method name.